modules/ROOT/nav.adoc

```diff
@@ -160,10 +163,11 @@
 **** xref:manage:security/console/tls-termination.adoc[TLS Termination]
 *** xref:manage:security/iam-roles.adoc[]
 ** xref:manage:tiered-storage-linux/index.adoc[Tiered Storage]
-*** xref:manage:tiered-storage.adoc[Tiered Storage]
-*** xref:manage:data-archiving.adoc[Data Archiving]
-** xref:manage:whole-cluster-restore.adoc[Whole Cluster Restore]
-** xref:manage:remote-read-replicas.adoc[Remote Read Replicas]
+*** xref:manage:tiered-storage.adoc[About Tiered Storage]
```

Comment:
This also contains 'how-to' info. I suggest giving this a title like 'Enable Tiered Storage'.